Makefile

```diff
@@ -9,7 +9,7 @@ GO := go
 .phony: .EXPORT_ALL_VARIABLES
 .EXPORT_ALL_VARIABLES:
 
-default: ensure build_go
+default: ensure_go build_go
```

Comment:
we renamed ensure -> ensure_go in this [commit](https://github.com/pulumi/esc-sdk/commit/13811222f44110d996525a030388106d3b45960f#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52) but need to change it here too